src/biome/text/pipeline.py

```diff
@@ -242,6 +241,20 @@ def train(
         finally:
             self.__restore_training_logging()
 
+    def __clone_pipeline(self):
+        """
+        Handles pipeline clone. Tries to apply a deepcopy as default and, if is not possible,
+        create pipelines from self instance type manually
+        """
+        try:
+            return copy.deepcopy(self)
```

Comment:
Do we need this deepcopy? why not always create a Pipeline from scratch?

Reply:
Not really, I will drop it.